WORKSPACE

```diff
@@ -258,6 +258,11 @@ local_repository(
     path = "third_party/fbgemm",
 )
 
+local_repository(
+    name = "unused_fmt",
```

Comment:
nit: to follow other naming conventions from this section, use `unused_ftm_bazel` as a name.